### ISSUE TITLE
CSS target property

### ISSUE THREAD

**Issue by anyotherway** [Fyrd/caniuse#3391]
CSS target property.
Since 2004, W3 Recommendation. 
NO browser support. 
EITHER base target="_blank" OR target=_self.
Please full demo, TARGET property will be useful: PUSH caniuse, push!

https://www.w3schools.com/cssref/css3_pr_target.asp
https://www.w3.org/TR/2004/WD-css3-hyperlinks-20040224/#target0

**Comment by Fyrd** [comment]
The latest version of this spec is at https://www.w3.org/TR/css3-hyperlinks/ and was discontinued. Seems clear no browsers are interested in implementing this.

**Comment by anyotherway** [comment]
Yes, we got that in my link to W3(0) I will find a machine reason we need it. And continuing my bitch at Apple… W3 Schools pages are more progressive and in touch that dusty W3 archives. Plenty of memory to handle a few more logical steps. Data attribution is 4D and it needs all the handles it can get. Period! My browser is not a 3D printer... YET!
